docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs

```diff
@@ -43,49 +51,68 @@ public static void Example()
-            // In this sample, CustomMappingEstimator is used to resize variable length vector to fixed length vector.
+            Console.WriteLine("Name: {0}, Type: {1}, Shape: (-1, {2})", "Features",
+                featuresType.ItemType.RawType, featuresType.Dimensions[0]);
+
+            var predictionType = (VectorDataViewType)schema["Prediction/Softmax"]
+                .Type;
+            Console.WriteLine("Name: {0}, Type: {1}, Shape: (-1, {2})",
+                "Prediction/Softmax", predictionType.ItemType.RawType,
+                predictionType.Dimensions[0]);
+
+            // The model expects the input feature vector to be a fixed length
+            // vector.
+            // In this sample, CustomMappingEstimator is used to resize variable
+            // length vector to fixed length vector.
             // The following ML.NET pipeline
             //      1. tokenzies the string into words, 
-            //      2. maps each word to an integer which is an index in the dictionary ('lookupMap'),
-            //      3. Resizes the integer vector to a fixed length vector using CustomMappingEstimator ('ResizeFeaturesAction')
+            //      2. maps each word to an integer which is an index in the
+            //         dictionary ('lookupMap'),
+            //      3. Resizes the integer vector to a fixed length vector using
+            //         CustomMappingEstimator ('ResizeFeaturesAction')
             //      4. Passes the data to TensorFlow for scoring.
-            //      5. Retreives the 'Prediction' from TensorFlow and put it into ML.NET Pipeline 
+            //      5. Retreives the 'Prediction' from TensorFlow and put it into
+            //         ML.NET Pipeline 
 
-            Action<IMDBSentiment, IntermediateFeatures> ResizeFeaturesAction = (i, j) =>
+            Action<IMDBSentiment, IntermediateFeatures> ResizeFeaturesAction = (i,
+                j) =>
             {
                 j.Sentiment_Text = i.Sentiment_Text;
                 var features = i.VariableLengthFeatures;
                 Array.Resize(ref features, MaxSentenceLength);
                 j.Features = features;
             };
 
-            var model = mlContext.Transforms.Text.TokenizeIntoWords("TokenizedWords", "Sentiment_Text")
-                .Append(mlContext.Transforms.Conversion.MapValue("VariableLengthFeatures", lookupMap,
-                    lookupMap.Schema["Words"], lookupMap.Schema["Ids"], "TokenizedWords"))
-                .Append(mlContext.Transforms.CustomMapping(ResizeFeaturesAction, "Resize"))
-                .Append(tensorFlowModel.ScoreTensorFlowModel("Prediction/Softmax", "Features"))
-                .Append(mlContext.Transforms.CopyColumns("Prediction", "Prediction/Softmax"))
-                .Fit(dataView);
-            var engine = mlContext.Model.CreatePredictionEngine<IMDBSentiment, OutputScores>(model);
+            var model = mlContext.Transforms.Text.TokenizeIntoWords(
```

Comment:
It would be better to try and align the Append methods, as well as keep the transform strings together. Something like:

        var model =
        mlContext.Transforms.Text.TokenizeIntoWords(
            "TokenizedWords",
            "Sentiment_Text")
        .Append(mlContext.Transforms.Conversion.MapValue(
            "VariableLengthFeatures",
            lookupMap,
            lookupMap.Schema["Words"],
            lookupMap.Schema["Ids"],
            "TokenizedWords"))
        .Append(mlContext.Transforms.CustomMapping(
            ResizeFeaturesAction,
            "Resize"))
        .Append(tensorFlowModel.ScoreTensorFlowModel(
            "Prediction/Softmax",
            "Features"))
        .Append(mlContext.Transforms.CopyColumns(
            "Prediction",
            "Prediction/Softmax"))
        .Fit(dataView);